api/lib/infrastructure/authentication.js

```diff
@@ -8,112 +8,143 @@ import { tokenService } from '../../src/shared/domain/services/token-service.js'
+    schemeName: schemes.jwt.name,
+    configuration: {
+      key: config.jwtConfig.poleEmploi.secret,
+      validate: validateClientApplication,
     },
-    {
-      name: 'jwt-parcoursup',
-      configuration: {
-        key: config.jwtConfig.parcoursup.secret,
-        validate: validateClientApplication,
-      },
+  },
+
+  jwtPixData: {
+    name: 'jwt-pix-data',
+    schemeName: schemes.jwt.name,
+    configuration: {
+      key: config.jwtConfig.pixData.secret,
+      validate: validateClientApplication,
     },
-  ],
+  },
 
-  defaultStrategy: 'jwt-user',
+  jwtParcoursup: {
+    name: 'jwt-parcoursup',
+    schemeName: schemes.jwt.name,
+    configuration: {
+      key: config.jwtConfig.parcoursup.secret,
+      validate: validateClientApplication,
+    },
+  },
 };
 
-function validateUser(decoded) {
-  return { isValid: true, credentials: { userId: decoded.user_id } };
-}
+const authentication = {
+  schemes,
+  strategies,
+  defaultStrategy: strategies.jwtUser.name,
+};
 
-function validateClientApplication(decoded) {
-  const application = find(config.apimRegisterApplicationsCredentials, { clientId: decoded.client_id });
-  if (!application) {
-    return { isValid: false, errorCode: 401 };
-  }
+async function validateUser(decodedAccessToken, { request, revokedUserAccessRepository }) {
+  // Only tokens including user_id are User Access Tokens.
+  // This is why applications Access Tokens are not subject to audience validation for now.
+  const userId = decodedAccessToken.user_id;
+  if (config.featureToggles.isUserTokenAudConfinementEnabled && userId) {
+    const revokedUserAccess = await revokedUserAccessRepository.findByUserId(userId);
+    if (revokedUserAccess.isAccessTokenRevoked(decodedAccessToken)) {
+      return { isValid: false };
+    }
 
-  if (decoded.scope !== application.scope) {
-    return { isValid: false, errorCode: 403 };
+    const audience = getForwardedOrigin(request.headers);
+    if (decodedAccessToken.aud !== audience) {
+      return { isValid: false };
+    }
   }
 
-  return { isValid: true, credentials: { client_id: decoded.clientId, scope: decoded.scope, source: decoded.source } };
+  return { isValid: true, credentials: { userId: decodedAccessToken.user_id } };
 }
 
-async function _checkIsAuthenticated(request, h, { key, validate }) {
-  const authorizationHeader = request.headers.authorization;
-  if (!authorizationHeader) {
-    return boom.unauthorized(null, 'jwt');
+async function validateClientApplication(decodedAccessToken) {
+  const application = find(config.apimRegisterApplicationsCredentials, { clientId: decodedAccessToken.client_id });
+  if (!application) {
+    return { isValid: false, errorCode: 401 };
   }
 
-  const accessToken = tokenService.extractTokenFromAuthChain(authorizationHeader);
-  if (!accessToken) {
-    return boom.unauthorized();
-  }
+  return {
+    isValid: true,
+    credentials: {
+      client_id: decodedAccessToken.clientId,
```

Comment:
C'était déjà une erreur présente avant, on retourne `clientId` au lieu de `client_id`
```suggestion
      client_id: decodedAccessToken.client_id,
```